_drafts/2016-09-20-iOS10-expands-autoplay-for-mobile.mdown

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: iOS 
```

Comment:
You can either come up with your own headline or what I do is "New <video> Policies for iOS | Webkit Blog"